style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 60px 10%;
+}
+
+nav a {
+  text-decoration: none;
+  font-size: 30px;
+  color: #6070ff;
+  font-weight: 700;
+}
+
+.content {
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
+  padding: 120px 10%;
+  background-image: url(images/image.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
Kindly use flexbox to lay out this section since it will provide you with a lot of options to align all your elements, plus, it also responsive.

Reply:
@jerryowusu  Thanks a lot for your review and suggestions.

I just summited a new commit with these changes. 

Looking forward.